### ISSUE TITLE
prisma auto-complete is not working

### ISSUE THREAD

**Issue by mittalyashu** [pnpm/pnpm#3852]
### Code to reproduce the issue:

```
const variable = await prisma.users.
```

### Actual behavior:

<img width="500" alt="CleanShot 2021-10-11 at 21 05 10@2x" src="https://user-images.githubusercontent.com/29014463/136817425-697fae2e-782b-4acf-a826-bfd99cc71c45.png">

### Additional information:

 - `node -v` prints: 16.3.0
 - Windows, macOS, or Linux?: MacOS

Prisma: https://www.prisma.io/

**Comment by mittalyashu** [comment]
> What editor do you use?

VS Code

**Comment by kieronjmckenna** [comment]
Did you find a solution to this?

**Comment by mittalyashu** [comment]
Yes, as if this comment, I am using v6.20.2 and it's seems to be working now.

Not sure if there was an actual fix released related to this issue or something weird going on my local machine. I'll leave the status of this issue to maintainers.

**Comment by zkochan** [comment]
Closing: this is not something pnpm can fix.

Editor autocompletion for a generated Prisma client is resolved by the TypeScript language service, not by pnpm. Prisma supports pnpm now; a current case belongs in Prisma's tracker.

If you still hit this on the latest pnpm, please comment with a reproduction and we will reopen.

---
Written by an agent (Claude Code, claude-opus-5).
